targets/mbedos5/jerryscript-mbed/jerryscript-mbed-event-loop/EventLoop.h

```diff
@@ -52,9 +52,6 @@ class EventLoop {
     Callback<void()> wrapFunction(jerry_value_t f) {
         MBED_ASSERT(jerry_value_is_function(f));
 
-        // not sure if this is necessary?
-        jerry_acquire_value(f);
```

Comment:
Should this not go in pair with `jerry_release_value` in `dropCallback`? If we wrote
```js
InterruptIn.raise(f());
InterruptIn.raise(null);
```
then would we not release `f` more than we acquired it?